sycl/include/sycl/detail/pi.h

```diff
@@ -654,6 +668,10 @@ constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_PRIORITY_LOW = (1 << 5);
 constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_PRIORITY_HIGH = (1 << 6);
 // clang-format on
 
+using pi_virtual_access_flags = pi_bitfield;
+constexpr pi_virtual_access_flags PI_VIRTUAL_ACCESS_FLAG_RW = (1 << 0);
+constexpr pi_virtual_access_flags PI_VIRTUAL_ACCESS_FLAG_READ_ONLY = (1 << 2);
```

Comment:
Any particular reason `PI_VIRTUAL_ACCESS_FLAG_READ_ONLY` is shifted by 2 rather than 1?

Reply:
Good catch! No, it was just stolen from `pi_mem_flags`. I will correct it.